app/src/main/java/com/example/series_collector/data/Series.kt

```diff
@@ -4,22 +4,25 @@ import androidx.room.ColumnInfo
 import androidx.room.Entity
 import androidx.room.PrimaryKey
 import com.google.firebase.firestore.DocumentId
-import java.util.Calendar
+import com.google.firebase.firestore.ServerTimestamp
+import java.util.*
 
 @Entity(tableName = "Series")
 data class Series(
     @PrimaryKey
     @DocumentId
     @ColumnInfo(name = "id")
-    val SeriesId: String,
+    val SeriesId: String = "",
```

Comment:
```suggestion
    val serisesId or id: String = "",
```
series class의 id property 이므로 id라고 명명하는게 조금더 좋을듯하고
`seriesId`를 사용하시더라도 camel case로 컨밴션을 통일해야됩니다.